<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addDialog</class>
 <widget class="QDialog" name="addDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="etuNimiLaatikko">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sukunimiLaatikko">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lohkoLaatikko">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lohkolappu">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>lohko</string>
   </property>
  </widget>
  <widget class="QLabel" name="sukunimilappu">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Sukunimi</string>
   </property>
  </widget>
  <widget class="QLabel" name="etulappu">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>etunimi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lahetaNappi">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>441</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>lähetä</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="arkkuBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Arkku</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="uurnaBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Uurna</string>
   </property>
  </widget>
  <widget class="QLabel" name="riviLappu">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Rivi</string>
   </property>
  </widget>
  <widget class="QLabel" name="paikkaLAppu">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Paikka</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="riviLaatikko">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="paikkaLaatikko">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="counterLaatikko">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="counterTeksti">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>lähetetty</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
